src/commands/ban_commands.py

```diff
@@ -15,15 +15,19 @@ def create_ban_commands(bot: Bot) -> None:
     @discord.app_commands.describe(
         user="User being banned",
         reason="Reason for ban",
+        delete_messages="Whether to delete the user's messages (default: False)",
     )
     async def ban(
         interaction: discord.Interaction,
         user: discord.Member,
         reason: str,
+        delete_messages: bool = False,  # Default to not deleting messages
```

Comment:
Doesn't the documentation say that to delete messages you need to say how many seconds worth of messages to delete?

Reply:
its hardcoded in to only be the same value. we will not give the option since there is really no need.

the only reason we would want to remove messages is for bots that comin. a week will be more than enough to delete spam messages